tests/app/main/helpers/test_frameworks.py

```diff
@@ -0,0 +1,294 @@
+# -*- coding: utf-8 -*-
+import pytest
+from nose.tools import assert_equal
+from app.main.helpers.frameworks import get_statuses_for_lot
+
+
+def get_lot_status_examples():
+    # limit, drafts, complete, declaration, framework
+    cases = [
+        # Lots with limit of one service
+        (
+            [True, 0, 0, None, 'open'],
+            []
+        ),
+        (
+            [True, 1, 0, None, 'open'],
+            [{
+                'title': u'Started but not complete',
+                'type': u'quiet'
+            }]
+        ),
+        (
+            [True, 0, 1, None, 'open'],
+            [{
+                'title': u'Marked as complete',
+                'hint': u'You can edit it until the deadline'
+            }]
+        ),
+        (
+            [True, 0, 1, 'complete', 'open'],
```

Comment:
What about the 'started' state for a declaration? Is that still a thing?  If so, could add that to the test cases.